frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -49,28 +49,44 @@ import { getStorages } from '../../selectors/selectors';
 const WINTOOLS_DISK_NAME = 'windows-guest-tools';
 
 const getContainerStorage = (
+  storageClassConfigMap: ConfigMapKind,
   diskType = DiskType.DISK,
   bus = DiskBus.VIRTIO,
-  image = '',
-): VMWizardStorage => ({
-  type: VMWizardStorageType.PROVISION_SOURCE_DISK,
-  disk: DiskWrapper.initializeFromSimpleData({
-    name: ROOT_DISK_NAME,
-    type: diskType,
-    bus,
-    bootOrder: 1,
-  }).asResource(),
-  volume: VolumeWrapper.initializeFromSimpleData({
-    name: ROOT_DISK_NAME,
-    type: VolumeType.CONTAINER_DISK,
-    typeData: { image },
-  }).asResource(),
-  editConfig: {
-    isFieldEditableOverride: {
-      source: false,
+  url = '',
+  size = '15Gi',
+): VMWizardStorage => {
+  const dataVolumeName = generateDataVolumeName(DUMMY_VM_NAME, ROOT_DISK_NAME);
+
+  return {
+    type: VMWizardStorageType.PROVISION_SOURCE_DISK,
+    disk: DiskWrapper.initializeFromSimpleData({
```

Comment:
lets not use the deprecated methods anymore

Reply:
fixed